test/test_example.py

```diff
@@ -1,14 +1,15 @@
 import argparse
+import glob
 import json
 import pathlib
 
 import pytest
-
 from otk import command
 
 
-@pytest.mark.parametrize("src_yaml", (pathlib.Path(__file__).parent / "data/base").glob("*.yaml"))
+@pytest.mark.parametrize("src_yaml", glob.glob("test/data/base/*.yaml"))
```

Comment:
Just as a note; this will require that tests are always ran from the root of the repository right? Perhaps repeating the above idiom or just wrapping the result of that iterator in a `[str(x) ...]` would be better?

Reply:
You're right!  I took my quick diff from the linked issue as is and forgot that we were doing it relative to the test file's path originally.

Fixing.